src/app/core/admin/admin-menu/admin-menu-item-details/admin-menu-item-details.component.ts

```diff
@@ -44,17 +44,24 @@ export class AdminMenuItemDetailsComponent implements OnInit {
   data = inject<{
     item: MenuItem;
     isNew?: boolean;
+    itemType?: string;
   }>(MAT_DIALOG_DATA);
 
   item: MenuItem | EntityMenuItem;
   availableRoutes: { value: string; label: string }[];
   isNew: boolean;
+  /** The type of item being edited (e.g., "Menu Item", "Shortcut") */
+  itemType: string;
```

Comment:
Do we still use the itemType property for anything important? Or can it be removed completely?

EDIT: I think now it can probably be merged with the isShortcut property - and we can give the remaining property a functional name, like "allowEntityLinks".